### PR TITLE
US-32: Dashboard - View task details

### PR DESCRIPTION
**Pull request by bbynosa**
- Initial layout to Task modal 
- Code cleanup on several components

### CODE REVIEW

File: src/routes/Dashboard/TaskViewEditModal.js [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+import { ReactComponent as MoveUpIcon } from "../../icons/TaskEditModal/move_up.svg";
+import { ReactComponent as MoveDownIcon } from "../../icons/TaskEditModal/move_down.svg";
+import { ReactComponent as MenuIcon } from "../../icons/TaskEditModal/menu.svg";
+import { ReactComponent as CloseIcon } from "../../icons/TaskEditModal/close.svg";
+
+import { ReactComponent as PriorityIcon } from "../../icons/icon_priority.svg";
+
+export default function TaskViewEditModal({
+  open,
+  closeTaskModal,
```
Comment (mlataza) [inline src/routes/Dashboard/TaskViewEditModal.js]: I suggest we rename this to 'onClose' as this is an event that should be handled by parent component.

File: src/routes/Dashboard/TaskViewEditModal.js [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+    data.due_date_day = dueDate.getDay();
+    data.due_date_month_name = dueDate
+      .toLocaleString("default", { month: "long" })
+      .substring(0, 3);
+
+    setTask(data);
+    setLoading(false);
+  };
+
+  const close = () => closeTaskModal();
```
Comment (mlataza) [inline src/routes/Dashboard/TaskViewEditModal.js]: I suggest we rename this function as 'handleClose'. Going forward, I suggest we use 'handle<Event>' naming convention for event handlers.

**Review by mlataza — CHANGES_REQUESTED** [review]

File: src/routes/Dashboard/TaskViewEditModal.js (hunk 2 above)
Comment (bbynosa) [reply src/routes/Dashboard/TaskViewEditModal.js]: Good suggestion

**Comment by bbynosa** [comment]
@mlataza - push 2 commits to resolve comments. can you please have a look again. thanks

**Comment by bbynosa** [comment]
current fixing merge conflicts

**Review by mlataza — APPROVED** [review]
LGTM

**Comment by bbynosa** [comment]
@mlataza - TY sir.

**Pull request merged**